src/hotspot/share/runtime/safepoint.cpp

```diff
@@ -162,11 +162,6 @@ bool SafepointSynchronize::thread_not_running(ThreadSafepointState *cur_state) {
   if (!cur_state->is_running()) {
     return true;
   }
-  LogTarget(Trace, safepoint) lt;
-  if (lt.is_enabled()) {
-    LogStream ls(lt);
-    cur_state->print_on(&ls);
-  }
```

Comment:
Why is this not useful for debugging? I can understand you might not need it for your profiling but UL is primarily about debugging assistance.